src/storm/models/ModelBase.h

```diff
@@ -99,6 +101,19 @@ class ModelBase : public std::enable_shared_from_this<ModelBase> {
      */
     virtual bool isSymbolicModel() const;
 
+    /*!
```

Comment:
Why do we need this for the supergeneric models? This sounds like something I would only want in DD models?

Reply:
The problem is that I can only downcast a `ModelBase` to a `storm::models::symbolic::Model<DdType, ValueType>` once I know the `DdType`.

Alternatively, we could introduce a `getModelRepresentation()` using [this](https://github.com/moves-rwth/storm/blob/ca1a439fa00b927fff563ae16dff8e7146429eba/src/storm/models/ModelRepresentation.h#L9) enum. Would you prefer that?